src/components/cvaa-overlay/main-captions_window.tsx

```diff
@@ -75,7 +75,7 @@ class MainCaptionsWindow extends Component<any, any> {
   render(props: any): VNode<any> {
     return (
       <div className={[style.overlayScreen, style.active].join(' ')}>
-        <h2 className={style.title}>
+        <h2 className={style.title} id="captionsDialogTitle">
```

Comment:
is there a way to avoid using id ?
id should be unique for the page, but we can have multiple players on a single page

Reply:
Unfortunately the dialog has a visible title and it must be referenced with aria-labelledby that must point to the title id. If the title would be missing we could've used just an aria-label property that doesn't use id.